conf/gir-pango-cairo.toml

```diff
@@ -0,0 +1,10 @@
+[options]
+work_mode = "sys"
+library = "PangoCairo"
+version = "1.0"
+min_cfg_version = "1.30"
+external_libraries = [
+    "GLib",
+    "Pango",
+    "Cairo",
```

Comment:
Seems this external cause problem as it don't have `cairo-sys` here

Reply:
It doesn't make sense? It's only using `sys` crates after all so adding `cairo-sys` in here couldn't work. Or did I miss something?

Reply:
Try manually edit pangocairo-sys/Cargo.toml for cairo-sys path. Seems it will be not replaced on update

Reply:
I did, it's not a problem anymore. ;)

Now remains two problems:

 1. FcFontMap (which seems correct to me as defined in the gir file...).
 2. cairo::FontType (it's an enum in cairo and gir doesn't seem to understand when I say that).

Reply:
Ok, number 2 is solved in a... veeeeeeery ugly way. But whatever, it works haha.

Reply:
Seems this type missed from pango.gir https://developer.gnome.org/pango/stable/PangoFcFontMap.html, so this gir looks as class is parent of yourself.
```
    <class name="FcFontMap"
           c:symbol-prefix="fc_font_map"
           c:type="PangoCairoFcFontMap"
           parent="Pango.FontMap"
           glib:type-name="PangoCairoFcFontMap"
           glib:get-type="pango_cairo_fc_font_map_get_type">
      <implements name="FontMap"/>
      <field name="parent_instance" introspectable="0">
        <type name="FcFontMap" c:type="PangoFcFontMap"/>
      </field>
```

Reply:
When I said this gir file was a huge mess, I wasn't kidding you know? :p